.github/workflows/build.yml

```diff
@@ -30,3 +30,8 @@ jobs:
       run: dotnet restore
     - name: Run build
       run: dotnet fake build -t Test
+    - name: Run Fsharplint on itself
+      run: |
+        cd ./src/FSharpLint.Console
+        dotnet run --framework net5.0 lint ../FSharpLint.Core/FSharpLint.Core.fsproj
```

Comment:
@Mersho what happens if you don't use the --framework flag here? This addition is kinda weird so it needs to be explained in the commit msg

Reply:
interesting!